src/redux/books/books.js

```diff
@@ -0,0 +1,17 @@
+const Books = () => (
+  <section>
+    <ul>
+      <li>Seven Habits of Highly Effective People ~ Unknown </li>
+      <button type="button">Remove</button>
+      <li>Rich Dad, Poor Dad ~ Unknown</li>
+      <button type="button">Remove</button>
+    </ul>
+    <form>
+      <input type="text" />
+      <input type="text" />
+      <input type="submit" value="Submit" />
+    </form>
+  </section>
+);
+
+export default Books;
```

Comment:
- [ ] All components should be in the `components` folder and responsible for rendering data. The `redux` folder is used for component state management and therefore should not host the component itself. Please move this component to the component folder